scale/scale_test_automation.go

```diff
@@ -0,0 +1,12 @@
+package scale
```

Comment:
packages should have a package godoc
https://github.com/golang/go/wiki/CodeReviewComments#package-comments